soc/riscv/riscv-privileged/wch_ch32v/Kconfig.series.qingke_v3

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2023 Chen Xingyu <hi@xingrz.me>
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_CH32V_QINGKE_V3A
```

Comment:
This config should be better to merge into `SOC_SERIES_CH32V`.
There are no settings where SOC_SERIES is divided and configured at this time.
And the core type can identify from the devicetree definitions.

Reply:
This config is used as a "shortcut" to select supported ISA extentions. In the further PR I'll submit the support for QingKe V4A and QingKe V4F series, which have different ISA extentions.

Reply:
No RISCV SoC makes such a "shortcut".
I think it's better to follow custom.